app/src/main/kotlin/com/hedvig/app/HedvigApplication.kt

```diff
@@ -22,11 +25,28 @@ open class HedvigApplication : Application() {
 
     whatsNewRepository.removeNewsForNewUser()
 
+    setupCustomerIo()
+
     registerActivityLifecycleCallbacks(ActivityChangeTracker())
 
     AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
   }
 
+  private fun setupCustomerIo() {
+    CustomerIO.Builder(
+      siteId = "id",
+      apiKey = "key",
+      appContext = this,
+    ).apply {
+      addCustomerIOModule(
+        ModuleMessagingPushFCM(),
+      )
+      setRequestTimeout(8000L)
+      setRegion(Region.US)
+      build()
+    }
+  }
```

Comment:
Might be a good idea to do what [DatadogInitializer](https://github.com/HedvigInsurance/android/blob/73e90644d4bd8e3442e1bca478c0ef565a57e836/datadog/src/main/kotlin/com/hedvig/android/datadog/DatadogInitializer.kt#L23) is doing to initialize itself, basically using androidx.startup. Can also be done that way so that the customer-io stuff can be in its own module. Can be done at a later PR though of course if we wanna do it.